### ISSUE TITLE
content_rating test not happy

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#4527]
Any chance getting this fixed for 0.15? It's a test so not a dangerous fix, just that it'd be nice to get rid of these inconclusive / random test failures.

Circle CI output...

```
  Scenario: User alters text feedback       # kalite/distributed/features/content_rating.feature:18
    Given I have filled out a feedback form # kalite/distributed/features/steps/content_rating.py:113
    Given I have filled out a feedback form # kalite/distributed/features/steps/content_rating.py:113 8.351s
      Assertion Failed: FAILED SUB-STEP: When I fill out the form
      Substep info: Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 108, in impl
          enter_star_ratings(context)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 159, in enter_star_ratings
          rate_id(context, id_)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_rating.py", line 179, in rate_id
          rating_el.click()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 65, in click
          self._execute(Command.CLICK_ELEMENT)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 402, in _execute
          return self._parent.execute(command, params)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 175, in execute
          self.error_handler.check_response(response)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 166, in check_response
          raise exception_class(message, screen, stacktrace)
      StaleElementReferenceException: Message: Element is no longer attached to the DOM
      For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
      Build info: version: '2.47.1', revision: '411b314', time: '2015-07-30 03:03:16'
      System info: host: 'chef', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'i386', os.version: '3.13.0-32-generic', java.version: '1.8.0_60'
      Driver info: driver.version: unknown
      Stacktrace:
          at <anonymous class>.fxdriver.cache.getElementAt (resource://fxdriver/modules/web-element-cache.js:9351)
          at <anonymous class>.Utils.getElementAt (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:8942)
          at <anonymous class>.fxdriver.preconditions.visible (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:9980)
          at <anonymous class>.DelayedCommand.prototype.checkPreconditions_ (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12626)
          at <anonymous class>.DelayedCommand.prototype.executeInternal_/h (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12643)
          at <anonymous class>.DelayedCommand.prototype.executeInternal_ (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12648)
          at <anonymous class>.DelayedCommand.prototype.execute/< (file:///tmp/anonymous6389913725208483987webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12590)
```

**Comment by MCGallaspy** [comment]
It had been working -- are you sure this isn't indicative of an actual error?

**Comment by MCGallaspy** [comment]
BTW, the links to the test run on sauce labs are helpful in determining whether there's a legitimate error or if our app is having a moment. Be sure to check those out & include them in cases like this where you think there is a smelly test. I can't find a build with this specific error.

**Comment by benjaoming** [comment]
It worked after rebuilding!

**Comment by benjaoming** [comment]
Here's another build with the same error:

https://circleci.com/gh/learningequality/ka-lite/1182

**Comment by MCGallaspy** [comment]
Okay, I'll try to figure out what's going on.

**Comment by benjaoming** [comment]
Awesome!

**Comment by benjaoming** [comment]
Just gonna wait it out for a couple of builds to see if it works, but I would think it does :)

**Comment by rtibbles** [comment]
Should be fixed in #4548
